src/console_cmds.cpp

```diff
@@ -614,6 +615,115 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	IniFile ini { group_names };
+	ini.LoadFromDisk(path, NO_DIRECTORY);
+	const IniGroup *ban_group = ini.GetGroup("bans", false);
+
+	if (ban_group == nullptr) return -1;
+
+	int loaded_count = 0;
+	for (const IniItem *item = ban_group->item; item != nullptr; item = item->next) {
+		// banned clients are added to global ban list when kicked or banned
+		NetworkServerKickOrBanIP(item->name.c_str(), true, nullptr);
+		loaded_count++;
+	}
+	return loaded_count;
+}
+
+/**
+* Loads a ban list from a user-specified file. The entries in the list are added to the current collection of banned entries.
+* @see ConBanList
+*/
+DEF_CONSOLE_CMD(ConBanListLoad)
+{
+	if (argc == 0) {
+		IConsoleHelp("Loads the IP's of banned clients from a file: Usage 'banlist_load <path_to_file>'");
```

Comment:
The deviates from the style used for the other help messages, and I think it should be IPs (plural) not IP's (possessive). Also make it clear that you do not replace the ban list, but rather add them to the ban list
`Load the IPs of banned clients from a file and add them to the ban list. Usage 'banlist_load <path_to_file>`